superset-frontend/src/components/Select/styles.tsx

```diff
@@ -241,9 +241,39 @@ export const DEFAULT_STYLES: PartialStylesConfig = {
   }),
 };
 
-const { ClearIndicator, DropdownIndicator, Option } = defaultComponents;
+const multiInputSpanStyle = css`
+  color: '#879399',
```

Comment:
if this color is part of the theme, https://github.com/apache-superset/superset-ui/blob/master/packages/superset-ui-core/src/style/index.ts, you should be able to reference it with `${({ theme }) => theme.colors...}`

Reply:
good to know. Unfortunately it doesn't appear to be in the theme.

Reply:
we could also add it to the theme if we expect reuse or want it to be themeable